src/test/components/NetworkChart.test.js

```diff
@@ -408,3 +413,32 @@ describe('EmptyReasons', () => {
     expect(container.querySelector('.EmptyReasons')).toMatchSnapshot();
   });
 });
+
+describe('Network Chart/tooltip behavior', () => {
+  beforeEach(addRootOverlayElement);
+  afterEach(removeRootOverlayElement);
+
+  it('shows a tooltip when the mouse hovers the line', () => {
+    const { rowItem, queryByTestId, getByTestId } = setupWithPayload(
+      'Load 100: https://test.mozilla.org',
+      {
+        type: 'Network',
+        URI: 'https://mozilla.org/img/',
+        id: 90001,
+        pri: 20,
+        count: 10,
+        status: 'STATUS_STOP',
+        startTime: 10,
+        endTime: 99,
+      }
+    );
+
+    expect(queryByTestId('tooltip')).toBeFalsy();
+    // React uses mouseover/mouseout events to implement mouseenter/mouseleave.
+    // See https://github.com/facebook/react/blob/b87aabdfe1b7461e7331abb3601d9e6bb27544bc/packages/react-dom/src/events/EnterLeaveEventPlugin.js#L24-L31
```

Comment:
spent about 2h on this... `fireEvent.mouseEnter` does that automatically, but it wasn't working when speciyfing the event "manually"...

Reply:
Ah woops.